src/types/pointer_constraints_v1.zig

```diff
@@ -0,0 +1,81 @@
+const wlr = @import("../wlroots.zig");
+
+const pixman = @import("pixman");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+const zpointer = wayland.server.zpointer;
```

Comment:
Hrm, this is a zig-wayland bug. We should be using the interface names to determine namespace not the protocol or file names. This belongs in `zwp`

Reply:
https://github.com/ifreund/zig-wayland/commit/5def9c58107c4803847e37511d16c15ff294b62d